kernel/src/expressions/scalars.rs

```diff
@@ -151,6 +151,16 @@ impl Scalar {
     pub fn is_null(&self) -> bool {
         matches!(self, Self::Null(_))
     }
+
+    /// Constructs a Scalar timestamp from an `i64` millisecond since unix epoch
+    pub fn timestamp_from_millis(millis: i64) -> DeltaResult<Self> {
```

Comment:
add a quick test?

Reply:
also maybe just pub crate for now?
```suggestion
    pub(crate) fn timestamp_from_millis(millis: i64) -> DeltaResult<Self> {
```

Reply:
Tried writing a test, but it felt like reimplementing it 
```rust
    #[test]
    fn test_millisecond_to_timestamp() {
        let timestamp = Duration::from_millis(123456789);
        let millis = timestamp.as_millis().try_into().unwrap();

        let Scalar::Timestamp(timestamp_micros) = Scalar::timestamp_from_millis(millis).unwrap()
        else {
            panic!("Expected output to be Timestamp");
        };

        let expected_micros: i64 = timestamp.as_micros().try_into().unwrap();
        assert_eq!(timestamp_micros, expected_micros)
    }
```